### PR TITLE
libbeat/scripts: parameterize all usages of go get

### PR DESCRIPTION
**Pull request by 7AC**

### CODE REVIEW

File: libbeat/scripts/Makefile [hunk 1]
```diff
@@ -30,6 +30,8 @@ COVERAGE_DIR=${BUILD_DIR}/coverage
 COVERAGE_TOOL=${BEAT_GOPATH}/bin/gotestcover
 COVERAGE_TOOL_REPO=github.com/elastic/beats/vendor/github.com/pierrre/gotestcover
 TESTIFY_TOOL_REPO=github.com/elastic/beats/vendor/github.com/stretchr/testify
+GOX=${BEAT_GOPATH}/bin/gox
+GOX_REPO=github.com/mitchellh/gox
```
Comment (ruflin) [inline libbeat/scripts/Makefile]: Do we need this as variable? I recommend to only use variables if we have multiple usages as our Makefile is already quite messy / long.

Comment (7AC) [inline libbeat/scripts/Makefile]: In this cases it was mainly for consistency with the other `go get`s.

**Comment by ruflin** [comment]
@7AC How should we move forward here? I prefer not to have variables when we don't need them but both works for me.

**Comment by 7AC** [comment]
I'll drop this one

**Pull request closed without merging**